flink-table/flink-table-uber-blink/pom.xml

```diff
@@ -113,6 +113,13 @@ under the License.
 									<include>org.apache.flink:flink-cep_${scala.binary.version}</include>
 								</includes>
 							</artifactSet>
+							<relocations>
+								<relocation>
+									<!-- icu4j's dependencies -->
```

Comment:
Why don't do this in `flink-table-planner-blink` module?

Reply:
because `flink-table-planner-blink` does not package `table-common` into jar. If we relocate `icu4j` here and users depends on `flink-table-planner-blink` in their project for developing. (that means `table-common` will indirectly dependent on) `ClassNotFoundException` will occur if user executes `TableResult.print()` in IDE.